doc/installation.rst

```diff
@@ -21,6 +21,20 @@ If you are on Debian or Ubuntu Linux we recommend you try
 
 .. _install-packages:
 
+
+Using Anaconda:
+===============
+
+On all platforms, you can use Anaconda_ to install Dipy. To do so issue the following two commands in a terminal::
+
+    conda config --add channels conda-forge
+    conda install dipy
+
+#. Some of the visualization methods require the VTK_ library and this can be installed separately (but only on Python 2)::
+
+    conda install vtk
+
```

Comment:
It might be that there are some advantages to not needing to specify the
channel to install from, but this has some advantages as well. Especially
with regards to maintenance of the recipe, because it's all done on the CI
services (see here for details: https://conda-forge.github.io/).

On Sat, May 21, 2016 at 4:45 PM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In doc/installation.rst
> <https://github.com/nipy/dipy/pull/1052#discussion_r64141989>:
>
> > @@ -21,6 +21,20 @@ If you are on Debian or Ubuntu Linux we recommend you try
> >
> >  .. _install-packages:
> >
> > +
> > +Using Anaconda:
> > +===============
> > +
> > +On all platforms, you can use Anaconda_ to install Dipy. To do so issue the following two commands in a terminal::
> > +
> > +    conda config --add channels conda-forge
> > +    conda install dipy
> > +
> > +#. Some of the visualization methods require the VTK_ library and this can be installed separately (but only on Python 2)::
> > +
> > +    conda install vtk
> > +
>
> Ah okay I saw your message. There is a PR that adds it soon.
> Do you know how easy it is to move from conda-forge to the actual conda?
> For example for vtk you don't need to do conda config --add channels
> conda-forge. You can just do conda install vtk. Is it like with
> neurodebian? For example you start adding packages in neurodebian and then
> they are moved when ready to debian?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/1052/files/d75ae785855126af805a6c47683115f737f80331#r64141989>
>